2023-01-09 00:13:09,538 INFO    MainThread:34564 [wandb_setup.py:_flush():68] Configure stats pid to 34564
2023-01-09 00:13:09,539 INFO    MainThread:34564 [wandb_setup.py:_flush():68] Loading settings from C:\Users\opben\.config\wandb\settings
2023-01-09 00:13:09,539 INFO    MainThread:34564 [wandb_setup.py:_flush():68] Loading settings from c:\Users\opben\OneDrive\Documents\Esh Release Radar\wandb\settings
2023-01-09 00:13:09,539 INFO    MainThread:34564 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-01-09 00:13:09,539 INFO    MainThread:34564 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-01-09 00:13:09,540 INFO    MainThread:34564 [wandb_init.py:_log_setup():476] Logging user logs to c:\Users\opben\OneDrive\Documents\Esh Release Radar\wandb\run-20230109_001309-msliynkg\logs\debug.log
2023-01-09 00:13:09,540 INFO    MainThread:34564 [wandb_init.py:_log_setup():477] Logging internal logs to c:\Users\opben\OneDrive\Documents\Esh Release Radar\wandb\run-20230109_001309-msliynkg\logs\debug-internal.log
2023-01-09 00:13:09,541 INFO    MainThread:34564 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001AC2CA05AB0>
2023-01-09 00:13:09,541 INFO    MainThread:34564 [wandb_init.py:init():516] calling init triggers
2023-01-09 00:13:09,541 INFO    MainThread:34564 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-01-09 00:13:09,541 INFO    MainThread:34564 [wandb_init.py:init():569] starting backend
2023-01-09 00:13:09,541 INFO    MainThread:34564 [wandb_init.py:init():573] setting up manager
2023-01-09 00:13:09,546 INFO    MainThread:34564 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn, using: spawn
2023-01-09 00:13:09,549 INFO    MainThread:34564 [wandb_init.py:init():580] backend started and connected
2023-01-09 00:13:09,550 INFO    MainThread:34564 [wandb_run.py:_label_probe_notebook():1116] probe notebook
2023-01-09 00:13:09,551 INFO    MainThread:34564 [wandb_run.py:_label_probe_notebook():1126] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-01-09 00:13:09,551 INFO    MainThread:34564 [wandb_init.py:init():658] updated telemetry
2023-01-09 00:13:09,602 INFO    MainThread:34564 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-01-09 00:13:13,682 WARNING MsgRouterThr:34564 [router.py:message_loop():77] message_loop has been closed
2023-01-09 00:13:13,682 WARNING MsgRouterThr:34564 [router.py:message_loop():77] message_loop has been closed
2023-01-09 00:13:14,651 ERROR   MainThread:34564 [wandb_init.py:init():1105] error
Traceback (most recent call last):
  File "c:\Users\opben\.conda\envs\esh-release-radar\lib\site-packages\wandb\sdk\wandb_init.py", line 1078, in init
    run = wi.init()
  File "c:\Users\opben\.conda\envs\esh-release-radar\lib\site-packages\wandb\sdk\wandb_init.py", line 697, in init
    result = handle.wait(
  File "c:\Users\opben\.conda\envs\esh-release-radar\lib\site-packages\wandb\sdk\lib\mailbox.py", line 259, in wait
    raise MailboxError("transport failed")
wandb.errors.MailboxError: transport failed
